examples/integrations/langchain/langchain.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>Please correct me if I am wrong but we need to create a lang smith dataset with this name:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">dataset = client.create_dataset(dataset_name, description="A sample dataset in LangSmith.")
</pre><p>Do we also need to create examples : </p>

<pre class="ql-syntax" spellcheck="false">client.create_examples(
    inputs=[
        {"postfix": "to LangSmith"},
        {"postfix": "to Evaluations in LangSmith"},
    ],
    outputs=[
        {"output": "Welcome to LangSmith"},
        {"output": "Welcome to Evaluations in LangSmith"},
    ],
    dataset_id=dataset.id,
) 
</pre><p>As per the docs: </p><p><a href="https://docs.smith.langchain.com" target="_blank">https://docs.smith.langchain.com/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I noticed in the written guide that users create a dataset through the UI ? Why is that if the user could just use the LS API ? as shown above?</p>